### DISCUSSION TITLE
Integration of translations

### DISCUSSION THREAD

**Discussion by Ilithy** [tanujnotes/Olauncher#158] [General]
Hi,
I would like to know if you had planned to add Olauncher's translations (done in PR), and is it useful or not to submit new ones? _(In my case, in French)_.

I hope not to bother you with this and thank you very much for your work on Olauncher.

**Comment by tanujnotes** [comment]
Hey, yes I do plan to add translations. Hopefully in the next update. Translation PRs are welcome. Thank you.

**Reply by Ilithy** [reply]
Excellent, I'll start quickly then, thanks for your work.

**Reply by Ilithy** [reply]
Done: https://github.com/tanujnotes/Olauncher/pull/160
